modules/oc-mirror-signature-verification.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-about-sig-mirroring-verification_{context}"]
+= Enabling signature verification for oc-mirror plugin v2
+
+Starting with {product-title} 4.19, oc-mirror plugin v2 supports signature verification, which is disabled by default. When enabled, the plugin verifies that container images match their signatures, ensuring they have not been altered and come from trusted sources. If a signature mismatch is detected, the mirroring workflow will fail.
+
+.Procedure
+
+. To enable signature verification for all images, run the following command:
++
+[source,terminal]
+----
+$ oc mirror --secure-policy=true
+----
+
+. To enable or disable signature verification for specific elements — such as a transport protocol, registry, namespace, or image — follow these steps:
+
+.. Create a `policy.json` file in either the `$HOME/.config/containers/` or `/etc/containers/` directory.
++
+[NOTE]
+====
+If your policy configuration file is located outside the default directory, you can specify its path by using the `--policy` flag with the `oc mirror` command.
+
+For more information, see link:https://github.com/containers/image/blob/main/docs/containers-policy.json.5.md[`containers-policy.json(5)`].
+====
+
+.. Define verification rules for the desired scope (for example, registry or image) using the appropriate policy configuration. You can set the verification requirement by specifying the desired rule under each element.
++
+.Example: Enable verification for a specific image only, and reject all other images.
```

Comment:
I would move "only" closer to the phrase it modifies, and remove the period at the end here:
```suggestion
.Example: Enable verification for only a specific image, and reject all other images
```